lib/Conversion/HWToBTOR2/HWToBTOR2.cpp

```diff
@@ -616,13 +584,10 @@ struct ConvertHWToBTOR2Pass
     genConst(value, w, op);
   }
 
-  // Wires can generally be ignored in bto2, however we do need
-  // to keep track of the new alias it creates
+  // Wires should have been removed in PrepareForFormal
   void visit(hw::WireOp op) {
-    // Retrieve the aliased operation
-    Operation *defOp = op.getOperand().getDefiningOp();
-    // Wires don't output anything so just record alias
-    setOpAlias(op, defOp);
+    op->emitError("Wires are not supported in btor!");
+    return signalPassFailure();
```

Comment:
Not blocking but it would be nicer to return LogicalResult::failure() instead of void in the future. Currently pass continues even after signalPassFailure();